src/lib/api/api.go

```diff
@@ -293,14 +294,18 @@ func (a *API) SendMatrix(w http.ResponseWriter, r *http.Request) {
 // of the react-UI.
 func (a *API) QuickTestInput(w http.ResponseWriter, r *http.Request) {
 	if err := r.ParseForm(); err != nil {
-		panic(err)
+		a.Message = err.Error()
+		a.send(w, a.marshalled())
```

Comment:
This seems like a repeated pattern. Can we make a func call for this and send the output? This func will be called across all the functions and hence, reduce the redundancy.

Reply:
Yeah sure!